### DISCUSSION TITLE
Add teams data for bedwars

### DISCUSSION THREAD

**Discussion by stravingo** [HypixelDev/PublicAPI#576] [Ideas]
Hi everybody!
I don't know if this feature has already been suggested, but it would be cool if the API could provide for Bedwars the composition of the teams (players ID, color of the teams) for the game of a given player ID.
Thanks!

**Comment by Clemintina** [comment]
This would open up abuse such as Party trackers, Allowing snipers to know if they're in a target's queue and potentially denicking ( if nicked ). This was an endpoint till 2020, although with less details known as the sessions endpoint.
